pkg/api/service/testing/make.go

```diff
@@ -48,6 +48,8 @@ func MakeService(name string, tweaks ...Tweak) *api.Service {
 	SetTypeClusterIP(svc)
 	// Default to 1 port
 	SetPorts(MakeServicePort("", 93, intstr.FromInt(76), api.ProtocolTCP))(svc)
+	// Default internalTrafficPolicy to "Cluster"
+	SetInternalTrafficPolicyCluster(svc)
```

Comment:
see https://github.com/kubernetes/kubernetes/pull/103467/files for a change to this pattern - let's not encode the value in the name unless it's a complex thing (like setting type implies multiple fields).

Follow this pattern instead:

```
// SetAllocateLoadBalancerNodePorts sets the allocate LB node port field.
func SetAllocateLoadBalancerNodePorts(val bool) Tweak {
	return func(svc *api.Service) {
		svc.Spec.AllocateLoadBalancerNodePorts = utilpointer.BoolPtr(val)
	}
}
```

Reply:
Thanks, updated!

Reply:
Last commit refactors these to return Tweak and moves it to the bottom of the file